inference/core/workflows/core_steps/models/foundation/florence2/v1.py

```diff
@@ -77,12 +77,13 @@
     },
     {"task_type": "detection-grounded-ocr", "florence_task": "<REGION_TO_OCR>"},
     {"task_type": "region-proposal", "florence_task": "<REGION_PROPOSAL>"},
+    {"task_type": "unstructured", "florence_task": ""},
```

Comment:
empty string here is not really easy to understand once appearing in if statement down the line